js/main.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+var ROOMS_RANGE = {
+  min: 1,
+  max: 1
+};
+var GUESTS_RANGE = {
+  min: 1,
+  max: 5
+};
+var PRICES_RANGE = {
+  min: 2000,
+  max: 7000
+};
+var ORDINATE = {
+  start: 130,
+  end: 630
+};
+var ABSCISSA = {
+  start: 0,
+  end: 1200
+};
+
+var PIN_WIDTH_HALF = 25;
+var PIN_HEIGHT = 70;
+var TITLES = ['Краски Отель', 'Отель Оазис Зип ', 'LUNA Hotel Krasnodar', 'D Hotel', 'Отель Терем', 'Home-otel', 'Отель Коржовъ', 'Hostel on Kostyleva', 'Golden Tulip Krasnodar'];
+var TYPES = ['palace', 'flat', 'house', 'bungalo'];
+var CHECKINS = ['12:00', '13:00', '14:00'];
+var CHECKOUTS = ['12:00', '13:00', '14:00'];
+var FEATURES = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
+var DESCRIPTION = 'Описание';
+var PHOTOS = ['http://o0.github.io/assets/images/tokyo/hotel1.jpg', 'http://o0.github.io/assets/images/tokyo/hotel2.jpg', 'http://o0.github.io/assets/images/tokyo/hotel3.jpg'];
+var HOTELS_COUNT = 8;
+
+var getRandomInRange = function (min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+function getShuffledArray(items) {
+  var newItems = items.slice();
+  for (var i = newItems.length - 1; i > 0; i--) {
+    var j = Math.floor(Math.random() * newItems.length);
+    var save = newItems[i];
+    newItems[i] = newItems[j];
+    newItems[j] = newItems[save];
+  }
+
+  return newItems;
+}
+
+var getRandomArrayCopy = function (items) {
+  var newItems = getShuffledArray(items);
+  var randomLength = getRandomInRange(0, newItems.length - 1);
+  return newItems.slice(0, randomLength + 1);
```

Comment:
Правильнее и понятнее так
```suggestion
  var randomLength = getRandomInRange(1, newItems.length);
  return newItems.slice(0, randomLength);
```